dali/test/python/test_video_pipeline.py

```diff
@@ -70,3 +77,10 @@ def test_stride_video_pipeline():
         pipe_out = pipe.run()
     del pipe
 
+def test_multi_gpu_video_pipeline():
+    gpus = get_gpu_num()
+    pipes = [VideoPipe(batch_size=BATCH_SIZE, data=VIDEO_FILES, device_id=d, num_shards=gpus) for d in range(gpus)]
+    for p in pipes:
+        p.build()
+        p.run()
+test_multi_gpu_video_pipeline()
```

Comment:
shouldn't this be run with nosetests? If so, remove the invocation

Reply:
Debug leftover